python/hail/methods/impex.py

```diff
@@ -1122,6 +1122,27 @@ def import_vcf(path, force=False, force_bgz=False, header_file=None, min_partiti
 @handle_py4j
 @typecheck(path=oneof(strlike, listof(strlike)))
 def index_bgen(path):
+    """Index .bgen files. :meth:`.HailContext.import_bgen` cannot run without these
```

Comment:
Index .bgen files as required by :meth:`.HailContext.import_bgen`.

Reply:
hail context is gone